pkg/miniogw/config.go

```diff
@@ -150,9 +151,11 @@ func (c Config) NewGateway(ctx context.Context,
 
 	segments := segment.NewSegmentStore(oc, ec, pdb, rs, c.MaxInlineSize)
 
-	// TODO(jt): wrap segments and turn segments into streams actually
-	// TODO: passthrough is bad
-	stream := streams.NewPassthrough(segments)
+	// segment size 64MB
+	stream, err := streams.NewStreamStore(segments, c.SegmentSize)
+	if err != nil {
+		return nil, err
+	}
 	obj := objects.NewStore(stream)
 
 	return NewStorjGateway(buckets.NewStore(obj)), nil
```

Comment:
i think here too; Im cautious of just returning nil cuz if there is an err for some odd reason in days to come, it'll be caught.